### PR TITLE
add redis and jmx mocks

### PR DESCRIPTION
**Pull request by pawel-kaminski-krk**

### CODE REVIEW

**Comment by pawel-kaminski-krk** [comment]
redis mock is changed to test containers

File: canaveral-mocks/canaveral-redis-mock/src/main/java/pl/codewise/canaveral/mock/redis/RedisMockProvider.java [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            hostProperty = key;
+            return this;
+        }
+
+        public RedisMockConfig registerPortUnder(String key) {
+            portProperty = key;
+            return this;
+        }
+
+        public RedisMockConfig overrideRedisVersion(String image) {
```
Comment (jakub-dzon) [inline canaveral-mocks/canaveral-redis-mock/src/main/java/pl/codewise/canaveral/mock/redis/RedisMockProvider.java]: There is nothing to override - there is no default value for `image`

Comment (pawel-kaminski-krk) [reply canaveral-mocks/canaveral-redis-mock/src/main/java/pl/codewise/canaveral/mock/redis/RedisMockProvider.java]: withRedisImage(String image)?

Comment (jakub-dzon) [reply canaveral-mocks/canaveral-redis-mock/src/main/java/pl/codewise/canaveral/mock/redis/RedisMockProvider.java]: Yes, that would be fine.

**Comment by jakub-dzon** [comment]
Will be released in version 1.4.1

**Comment by pawel-kaminski-krk** [comment]
very well

**Pull request closed without merging**